test/distributed/test_distributed.py

```diff
@@ -2135,6 +2135,56 @@ def test_DistributedDataParallel_SyncBatchNorm_2D_Input(self):
             )
             self._barrier()
 
+    @unittest.skipIf(BACKEND != 'nccl' and BACKEND != 'gloo',
```

Comment:
@unittest.skipIf(BACKEND != 'nccl' ....), syncNorm is only supported for nccl

Reply:
Yeah, you're right. But since the original SyncBatchNorm test function `test_DistributedDataParallel_SyncBatchNorm()` uses this flag (and I'm not sure why), I might just continue the tradition. :)

About the failing test on CI, it seems like it's related to some networking issues (weird time out) and probably not related to the code.